src/components/Layout/AppBar/index.tsx

```diff
@@ -8,6 +8,7 @@ import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
 import NotificationsIcon from '@material-ui/icons/Notifications';
 import { TranslationContextProps } from 'ra-core';
 import { QilinLogo } from 'assets/icons';
+import LocaleSwitcher from 'components/LocaleSwitcher';
```

Comment:
```suggestion
import  { LocaleSwitcher } from 'components';
```
Вроде же так договорились.